### ISSUE TITLE
option service seems to be broken in `check-consul-service-health.rb`

### ISSUE THREAD

**Issue by majormoses** [sensu-plugins/sensu-plugins-consul#20]
I was going to start trying to expand this check to allow specifying a minimum threshold. When I started looking at the code and what it was producing I was a little confused. I backed out all my changes and saw the same behavior. Here is what I see:

```
$ ./bin/check-consul-service-health.rb -s es-app-master --consul http://10.55.250.176:8500 --fail-if-not-found
CheckConsulServiceHealth CRITICAL: Could not find checks for service es-app-master
```

![image](https://cloud.githubusercontent.com/assets/3145127/18686892/8205997c-7f32-11e6-96a7-ac9a905ad940.png)

so I tried adding a little bit of print code to see what was going on:

```
$ git diff
diff --git a/bin/check-consul-service-health.rb b/bin/check-consul-service-health.rb
index 55cb317..f0ddefc 100755
--- a/bin/check-consul-service-health.rb
+++ b/bin/check-consul-service-health.rb
@@ -78,6 +78,9 @@ class CheckConsulServiceHealth < Sensu::Plugin::Check::CLI
     criticals  = false
     checks     = {}

+    service_data = acquire_service_data
+    p "service_data: #{service_data}"
+
     # Process all of the nonpassing service checks
     acquire_service_data.each do |d|
       found       = true
```

to which is was very suprised to see:

```
13:04:04 [ prod-west.cloudcruiser.com ] [  ] [ ~/projects/personal/sensu-plugins-consul (git, master, *) ]
$ ./bin/check-consul-service-health.rb -s es-app-master --consul http://10.55.250.176:8500 --fail-if-not-found
"service_data: []"
CheckConsulServiceHealth CRITICAL: Could not find checks for service es-app-master
```

I verified that using the --all works and contains es-app-master

When I look at what is happening in irb this is what I see:

```
irb(main):006:0> Diplomat.configure do |dc|
irb(main):007:1* dc.url = 'http://10.55.250.176:8500'
irb(main):008:1> end
=> "http://10.55.250.176:8500"
irb(main):009:0> Diplomat::Health.checks('es-app-master')
=> []
```

I need to investigate more but it seems to likely be a bug in diplomat gem...Unfortunately I got pulled into another project so I might not have the time to track this down all the way but wanted to make sure that I had my notes down even for myself to pick back up if someone does not beat me to this.

**Comment by wg-tsuereth** [comment]
FWIW, I haven't encountered a problem like this.  Running with Diplomat 0.14.0 (which I suppose is quite old by now) and hitting Consul 0.7.0 (recently upgraded from 0.6.3).  Are you certain that the name `es-app-master` matches the service name registered in Consul?

**Comment by majormoses** [comment]
@wg-tsuereth hmm I wonder if that endpoint is available in 0.6.3 (I really need to upgrade) as I had checked that endpoint based on my investigation in diplomat and didn't see the check. I will try to squeeze some time in this week to run this to ground
